modules/exploits/linux/http/vmware_view_planner_4_6_uploadlog_rce.rb

```diff
@@ -0,0 +1,143 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  # "Shotgun" approach to writing JSP
+  Rank = ManualRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'VMware View Planner Unauthenticated Log File Upload RCE',
+        'Description' => %q{
+          This module exploits an unauthenticated log file upload within the
+          log_upload_wsgi.py file of VMWare View Planner 4.6 prior to 4.6
+          Security Patch 1.
+
+          Successful exploitation will result in RCE as the apache user inside
+          the appacheServer docker container.
+        },
+        'Author' => [
+          'Mikhail Klyuchnikov', # Discovery
+          'wvu', # Analysis and PoC
+          'Grant Willcox' # Metasploit Module
+        ],
+        'References' => [
```

Comment:
Feel free to link directly to https://attackerkb.com/assessments/fc456e03-adf5-409a-955a-8a4fb7e79ece if you need a source.

Reply:
Thanks will add this to the list of links so long now 👍

Reply:
Resolved with https://github.com/rapid7/metasploit-framework/pull/14875/commits/6293581363766f5079c7c295b6d20e89ba0c2e74